spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 
   # Stub all GitHub API requests so they come back empty.
   config.before(:each) do
-    stub_request(:get, /api.github.com/)
+    stub_request(:get, /api\.github.com/)
```

Comment:
Should both periods be made literal?

Reply:
yep I cannot read 😁. But I'll go over my other PR first, the one I dropped mid-way because it will help the CI!